android/src/facebook/LoginButtonView.java

```diff
@@ -48,19 +48,19 @@ public LoginButtonView(TiViewProxy proxy)
 	public void processProperties(KrollDict props)
 	{
 		super.processProperties(props);
-		Log.d(TAG, "[VIEW LIFECYCLE EVENT] processProperties " + props);
+
 		if (props.containsKey("publishPermissions")) {
-			Object value = props.get("publishPermissions");
-			if (value instanceof Object[]) {
-				String[] publishPermissions = TiConvert.toStringArray((Object[]) value);
-				loginButton.setPublishPermissions(Arrays.asList(publishPermissions));
-			}
+			Log.w(TAG,
+				  "The \"publishPermissions\" property has been deprecated in favor of the \"permissions\" property");
```

Comment:
Logic for `publishPermissions ` and `readPermissions` needs re-adding. Since these changes remove these properties and not deprecate them.

Reply:
@garymathews They are actually removed in the native SDK already (no-op functions natively), but it can be added again I think. On iOS (merged on master), it's the same behavior as here right now I think.

Reply:
We can just pass to
```Java
loginButton.setPermissions(Arrays.asList(...));
```
For backwards compatibility right?

Reply:
Updated PR to merge deprecated `publishPermissions` and `readPermissions` property arrays to the main non-deprecated `permissions` collection/method. Kept the logged warnings. There's slight duplication in the module's Java `propertyChanged()` method, but I think we can live with it for now... until we remove them later.